websites/J/Jellyfin/presence.ts

```diff
@@ -701,12 +697,18 @@ async function setDefaultsToPresence(): Promise<void> {
   if (presenceData.smallImageText) {
     delete presenceData.smallImageText;
   }
+  /*
   if (presenceData.startTimestamp) {
     delete presenceData.startTimestamp;
   }
-  if (presenceData.endTimestamp) {
+  if (presenceData.endTimestamp && isNaN(presenceData.endTimestamp)) {
     delete presenceData.endTimestamp;
   }
+  */
+
+  // set default values, required for Math.floor on devHelper.ts
+  presenceData.startTimestamp = 0;
+  presenceData.endTimestamp = 0;
```

Comment:
make sure to delete them if you dont set them

Reply:
I was previously deleting them, but if they are unset I get `NaN` on those fields.

I revert is to the previous code, deleting the values